src/Form/Validator/ErrorElement.php

```diff
@@ -174,10 +176,11 @@ public function getSubject()
      * @param string|array $message
      * @param array        $parameters
      * @param null         $value
+     * @param string       $translationDomain
      *
      * @return ErrorElement
      */
-    public function addViolation($message, $parameters = [], $value = null)
+    public function addViolation($message, $parameters = [], $value = null, $translationDomain = null)
```

Comment:
This is a BC-break for extending classes.

Reply:
If i add a new public method who give the possibility to set a translation domain is better ?

Reply:
if not in a is possible to apply the change in a new version ?

Reply:
https://3v4l.org/mk3eT

You can only do this kind of change on a major version.

Reply:
Or you could simulate a new argument and get it thanks to `func_get_args()`

Reply:
https://3v4l.org/InHLB